config/targets.js

```diff
@@ -1,17 +1,21 @@
 'use strict';
 
+// Production detection seems to be slightly
+// unreliable. Just comment out 'ie 11' when
+// developing to make it easier to trace errors.
 const browsers = [
   'last 1 Chrome versions',
   'last 1 Firefox versions',
-  'last 1 Safari versions'
+  'last 1 Safari versions',
+  'ie 11'
 ];
 
-const isCI = !!process.env.CI;
-const isProduction = process.env.EMBER_ENV === 'production';
-
-if (isCI || isProduction) {
```

Comment:
I wonder why this didn't work? `EMBER_ENV` _is_ set to "production" even on staging.

Reply:
Maybe these ENV variables aren't coming in here as we think they should?